docs/changes.rst

```diff
@@ -42,6 +42,13 @@ Released: not yet
 * Added a 'pywbemlistener' command for running and managing WBEM listeners.
   (issues #430, #479, #948)
 
+* Implement server schema command that returns information about the schema
+  for each namespace including: 1. The namespace, 2. schema version, 3. whether
+  any classes in the schema/namespace are experimental, 4) the number of
+  classes in this schema, and 5. the DMTF schemas (characters before the `_` in
+  the namespace). (see issue #444)
```

Comment:
Move item 5 to the front, because the name of the schema is the most important information. Also, I suggest talking about "schema name". The term "DMTF schema" is incorrect because this is about general schemas.

Reply:
DONE